doc/11-Concepts.md

```diff
@@ -0,0 +1,77 @@
+# Views
+
+## List Items
+The lists are one of the most prominent views for the monitoring. In the default view they follow this style:
+![DefaultAnatomy](res/ListItemAnatomy.jpg "A list item stripped down to the skeleton")
+
+### Visual
+Each element has a so-called visual. It's purpose is to highlight certain elements in long lists and give an intuitive overview about the underlying objects' condition. 
+In the case of host or service lists, for example, the state is displayed. This makes it immediately clear if and where there are problems that need to be taken care of, immediately from the overview.
+
+### Title
+The title briefly describes the state of the list element as an addition to the visual. For example, it contains the information that a host is currently DOWN. While the visual gives an intuitive impression, the title explains what exactly happened.
+
+### Meta
+The meta area is for additional information, usually displays of time. In the Host and Service Lists, you can see how long the object has been in the corresponding state.
+
+### Caption
+The caption area contains detailed information about the list item. In the case of hosts and services this would be the plugin output. For comments and downtimes the comment texts of the user are displayed here. To make the default list view compact and consistent, long texts are truncated to a maximum of two lines.
```

Comment:
Replace `plugin output` with `check output`. This aligns with `check source` and everything else. Icinga's check results are not necessarily populated from plugins, and new users may not understand the term at this point.

Reply:
good point, thanks!